### DISCUSSION TITLE
Puppeteer and cookieconsent

### DISCUSSION THREAD

**Discussion by massimo-cassandro** [orestbida/cookieconsent#741] [Q&A] [answered]
I am trying to get a list of a website that uses cookieconsent with puppeter, but the cookieconsent banner does not load.
I have tried with other cookie banner managers and they seem to work fine.

I've disabled the headless option to watch the browser, and the cookie banner doesn't appear:

```
import puppeteer from 'puppeteer';

const browser = await puppeteer.launch({
  headless: false,
});
const page = await browser.newPage();
await page.goto(cfg.url, {waitUntil : 'networkidle2' });
```

Any ideas?

Thanks

**Comment by orestbida** [comment, marked as answer]
Try setting the [`hideFromBots`](https://cookieconsent.orestbida.com/reference/configuration-reference.html#hidefrombots) option to false.

**Comment by massimo-cassandro** [comment]
it works!
Thanks
